### PR TITLE
Docs: Google Spreadsheets datasource

### PR DESCRIPTION
**Pull request by alexanderlz**
updated docs

### CODE REVIEW

File: _posts/2015-02-19-1-managing-datasources.md [hunk 1]
```diff
@@ -89,6 +89,21 @@ Notes:
 1. To obtain BigQuery credentials follow the guidelines at: https://developers.google.com/bigquery/authorization#service-accounts
 2. You need to install the `google-api-python-client`, `oauth2client` and `pyopenssl` packages (PyOpenSSL requires `libffi-dev` and `libssl-dev` packages), as they are not included in the `requirements.txt` file.
 
+### Google Spreadsheets
+
+* **Type**: google_spreadsheets
+* **Options**:
+	* Credentials filename (credfilePath) (mandatory)
+* **Options string format (for v0.5 and older)**: {"credfilePath" : "/somewhere/23fjkfjdsfj21312-privatekey.p12"}
```
Comment (arikfr) [inline _posts/2015-02-19-1-managing-datasources.md]: No need for this, it's supported from v0.6.3 and above anyway :)...

Comment (arikfr) [inline _posts/2015-02-19-1-managing-datasources.md]: But you should add a notice saying that (that it's supported from v0.6.3).

File: _posts/2015-02-19-1-managing-datasources.md [hunk 2]
```diff
@@ -89,6 +89,21 @@ Notes:
...
+### Google Spreadsheets
+
+* **Type**: google_spreadsheets
+* **Options**:
+	* Credentials filename (credfilePath) (mandatory)
+* **Options string format (for v0.5 and older)**: {"credfilePath" : "/somewhere/23fjkfjdsfj21312-privatekey.p12"}
+
+Notes:
+
+1. To obtain Google ServiceAccount credentials follow the guidelines at: https://developers.google.com/console/help/new/#serviceaccounts
```
Comment (arikfr) [inline _posts/2015-02-19-1-managing-datasources.md]: They need to download the JSON credentials file?

Comment (alexanderlz) [inline _posts/2015-02-19-1-managing-datasources.md]: That's the final part of the guidelines, a json file

Comment (arikfr) [inline _posts/2015-02-19-1-managing-datasources.md]: The final part mentions both p12 file and JSON... 

Also don't forget to update here the name of the param.

**Pull request merged**